Mathlib/Algebra/IndicatorFunction.lean

```diff
@@ -584,8 +573,10 @@ theorem mulIndicator_diff (h : s ⊆ t) (f : α → G) :
 #align set.mul_indicator_diff Set.mulIndicator_diff
 #align set.indicator_diff' Set.indicator_diff'
 
-theorem indicator_diff {G : Type*} [AddGroup G] {s t : Set α} (h : s ⊆ t) (f : α → G) :
-    indicator (t \ s) f = indicator t f - indicator s f := by rw [indicator_diff' h, sub_eq_add_neg]
+@[to_additive indicator_diff]
+theorem mulIndicator_diff' (h : s ⊆ t) (f : α → G) :
```

Comment:
Could you do the same for `compl`?
Also, it's a bit annoying if the primes in the name are mixed up (IIUC with your change we would have `mulIndicator_diff <-> indicator_diff'` and `mulIndicator_diff' <-> indicator_diff`), should we make sure that they match up (in a follow up PR since that's probably going to touch a lot of files) or is it expected that the "default" lemma is different?

Reply:
It seems that we mostly use `a * b⁻¹` for groups, `a / b` for fields, and `a - b` for additive groups.

Reply:
Okay that makes sense. Let's keep it like that then.